tskv/src/memcache.rs

```diff
@@ -394,6 +398,10 @@ impl MemCache {
     }
 
     pub fn write_group(&self, sid: SeriesId, seq: u64, group: RowGroup) -> Result<()> {
+        self.max_ts.store(
```

Comment:
The modify of `cache - max_ts` is too early that before fallible check and the real write to cache. If the check or write fails, what to do next?